service/sqs/cust_checksum_validation.go

```diff
@@ -0,0 +1,214 @@
+package sqs
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	sqstypes "github.com/aws/aws-sdk-go-v2/service/sqs/types"
+	"github.com/aws/smithy-go/middleware"
+)
+
+//********************************
+// SendMessage checksum validation
+//********************************
```

Comment:
minor nit on the styling of the comments. Even if its an internal function should we stick to the typical expected formatting as if it was a public doc string?

Reply:
Will update this. Used these as section separators, but regular doc string probably are better